content/applications/general/users/access_rights.rst

```diff
@@ -2,66 +2,148 @@
+     receive access to the *Website/Restricted Editor* group as well.
+
+- :guilabel:`Menus` tab: defines which menus/models the group can have access to.
+- :guilabel:`Views` tab: lists which views in Odoo the group has access too.
+- :guilabel:`Access Rights` tab: lists the first level of rights (models) that this group has access
+  rights to. In this tab, the :guilabel:`Model` column represents the common name of the menu/model,
+  and the :guilabel:`Name` column represents the technical name given to the model. For each model,
+  enable the following options as appropriate:
+
+  - :guilabel:`Read`: users can see the object's existing values.
+  - :guilabel:`Write`: users can edit the object's existing values.
+  - :guilabel:`Create`: users can create new values for the object.
+  - :guilabel:`Delete`: users can delete values for the object.
+
+  .. tip::
+     First try searching for the common name of the model in the drop-down menu of the
+     :guilabel:`Model` column. The :guilabel:`Model` technical name can be found by expanding the
+     model common name by clicking the :guilabel:`expand` button.
+
+     The model technical name can also be accessed in :ref:`developer mode <developer-mode>`. On a
+     form navigate to any field, and hover over the field name. A box of backend information reveals
+     itself with the specific Odoo :guilabel:`Object` name in the backend. This is the technical
+     name of the model that should be added.
+
+- :guilabel:`Record Rules`: lists the second layer of editing and visibility rights. Record Rules
+  overwrite, or refine, the group's *Access Rights*. For each rule, choose values for the following
+  options:
+
+  - :guilabel:`Apply for Read`.
+  - :guilabel:`Apply for Write`.
+  - :guilabel:`Apply for Create`.
+  - :guilabel:`Apply for Delete`.
+
+  .. important::
+     Record rules are written using a *Domain*, or conditions that filter data. A domain expression
+     is a list of such conditions. Users without knowledge of domains and domain expressions should
+     consult an Odoo Business Analyst or the Odoo Support Team before making changes.
+
+Superuser mode
+==============
+
+*Superuser mode* allows the user to bypass record rules and access rights. To activate *Superuser
+mode*, first, activate :ref:`developer mode <developer-mode>`. Then navigate to the :guilabel:`🪲
+(debug)` menu in the top banner. The :guilabel:`🪲 (debug)` menu can be accessed by clicking on the
+:guilabel:`🪲 (debug)` icon. Finally, towards the bottom click :guilabel:`Become Superuser`.
 
 .. important::
-   Making changes in access rights can have a big impact on the database. For this reason, we
-   recommend you to contact your Odoo Business Analyst or our Support Team, unless you have
-   knowledge about Domains in Odoo.
+   Only users with :guilabel:`Settings` access for the Administration section of the *Access Rights*
```

Comment:
```suggestion
   Only users with *Settings* access for the *Administration* section of the *Access Rights* (in their user profile)
```
using italics here because they are being referenced and aren't **currently** being shown on the screen/UI